scss/atoms/_tables.scss

```diff
@@ -0,0 +1,69 @@
+// ==========================================================================
+// Typography
+// ==========================================================================
+
+table {
+	width: 100%;
+	margin-bottom: em( 20px, 16px );
+	font-size: em( 16px );
+	text-align: left;
+}
+
+thead {
+	border-bottom: 1px solid #eee;
+
+	> tr > th {
+		border-top: 0;
+	}
+}
+
+td,
+th {
+	border-top: 1px solid #eee;
+	padding: em( 12px, 16px );
+	white-space: nowrap;
+}
+
+th {
+	padding: em( 12px, 12px );
```

Comment:
In that case is there really a point in combining the `td` and `th` rules here since they only actually share 2 properties top border and white space. Maybe just add those 2 properties here to the `th` rule and it will make it more clear whats going on to avoid confusion.